### PR TITLE
CSharpConvertAutoPropertyToFullProperty... handles init accessor

### LINKED ISSUE #48133: Support init accessors in CSharpConvertAutoPropertyToFullPropertyCodeRefactoringProvider
Currently only get; set; are supported.

https://github.com/dotnet/roslyn/blob/f6c95ab082171ef9a9f2c6114b003eea1df11842/src/Features/CSharp/Portable/ConvertAutoPropertyToFullProperty/CSharpConvertAutoPropertyToFullPropertyCodeRefactoringProvider.cs#L70-L73

### PR DESCRIPTION
**Pull request by louis-z**
Fixes #48133

### CODE REVIEW

**Comment by louis-z** [comment]
I hope I'm not being overly simplistic here, but
- the fix appears to be 1 line of code, and
- I only modified a single test, to validate that the code fixer behaves the same way whether dealing with `set` or `init`

LMK if I should do more.

File: src/EditorFeatures/CSharpTest/ConvertAutoPropertyToFullProperty/ConvertAutoPropertyToFullPropertyTests.cs [hunk 1]
```diff
@@ -22,32 +22,34 @@ public partial class ConvertAutoPropertyToFullPropertyTests : AbstractCSharpCode
         protected override CodeRefactoringProvider CreateCodeRefactoringProvider(Workspace workspace, TestParameters parameters)
             => new CSharpConvertAutoPropertyToFullPropertyCodeRefactoringProvider();
 
-        [Fact, Trait(Traits.Feature, Traits.Features.ConvertAutoPropertyToFullProperty)]
-        public async Task SimpleAutoPropertyTest()
+        [Theory, Trait(Traits.Feature, Traits.Features.ConvertAutoPropertyToFullProperty)]
+        [InlineData("set")]
+        [InlineData("init"), WorkItem(48133, "https://github.com/dotnet/roslyn/issues/48133")]
```
Comment (CyrusNajmabadi) [inline src/EditorFeatures/CSharpTest/ConvertAutoPropertyToFullProperty/ConvertAutoPropertyToFullPropertyTests.cs:27]:
```suggestion
        [InlineData("set"), InlineData("init")]
        [WorkItem(48133, "https://github.com/dotnet/roslyn/issues/48133")]
```

**Comment by CyrusNajmabadi** [comment]
> 'm not being overly simplistic here, but

Nope!  this looks just right to me :)

**Review by CyrusNajmabadi — APPROVED** [review]

**Review by CyrusNajmabadi — APPROVED** [review]

**Comment by CyrusNajmabadi** [comment]
Thanks for the contribution!

**Pull request merged**